manager/integration/tests/common.py

```diff
@@ -1571,6 +1571,7 @@ def volume_write(v, start, data):
 
 
 def dev_write(dev, start, data):
+    data = bytes(data, encoding='utf-8')
```

Comment:
Same here.

Reply:
In `fdev.write(data)` function expects the `data` to be in a bytes-like object or this error occurs:

> TypeError: a bytes-like object is required, not 'str'